pkg/api/defaults.go

```diff
@@ -69,7 +69,7 @@ func DefaultOperatorFlags() OperatorFlags {
 		"aro.azuresubnets.serviceendpoint.managed": flagTrue,
 		"aro.banner.enabled":                       flagFalse,
 		"aro.checker.enabled":                      flagTrue,
-		"aro.dnsmasq.enabled":                      flagTrue,
+		"aro.dnsmasq.enabled":                      flagFalse,
```

Comment:
I think this will break all the existing clusters that use egress lockdown.  

I was hoping more for an approach where there's a new feature flag and a new file that's laid down, either within the dnsmasq controller, or a separate controller.